package.json

```diff
@@ -43,7 +43,8 @@
     "tar-stream": "^1.1.1",
     "taskcluster-client": "^0.19.4",
     "url-join": "0.0.1",
-    "xml2js": "^0.4.4"
+    "xml2js": "^0.4.4",
+    "taskcluster-cli": "^0.5.3"
```

Comment:
Is this added so taskcluster-cli is available when running the cache script? If so, I'm not sure if this would give the desired results.  `taskcluster` should be in your path (i.e. /usr/local/bin/taskcluster) for this to run, but the package dependencies listed here are installed under ./node_modules/taskcluster-cli and is probably not part of most people's path.